rust/stracciatella/src/schemas/yaml/items.schema.yaml

```diff
@@ -1,142 +1,29 @@
-$schema: http://json-schema.org/draft-04/schema#
+$schema: http://json-schema.org/schema#
 type: array
 title: Items
 description: |
   **Limitations**: Some items are still referenced in source code, so they should not be changed.
 items:
-  title: Item
-  description: |
-    An item defines item properties of an item available in the game.
+  allOf:
+    - title: Base Item
+      $ref: types/base-item.schema.yaml
+    - title: Item
+      description: |
+        An item defines item properties of an item available in the game.
 
-    Note that weapons and magazines are not part of this list, but have their own definitions.
-  type: object
-  properties:
-    itemIndex:
-      title: Item Index
-      $ref: types/uint16.schema.yaml
-    internalName:
-      title: Internal name
-      $ref: types/id.schema.yaml
-    shortName:
-      title: Short name
-      type: string
-    name:
-      title: Name
-      type: string
-    description:
-      title: Description
-      type: string
-    usItemClass:
-      title: Item class
-      $ref: types/uint32.schema.yaml
-    ubClassIndex:
-      title: Item class index
-      $ref: types/uint8.schema.yaml
-    ubCursor:
-      title: Cursor
-      $ref: types/uint8.schema.yaml
-    inventoryGraphics:
-      title: Inventory Graphics
-      description: Graphics that are used in the inventory interface for this item
+        Note that weapons and magazines are not part of this list, but have their own definitions.
       type: object
       properties:
-        small:
-          title: Small
-          description: Small graphics. Used e.g. in strategic & tactical inventory
-          $ref: types/graphic.schema.yaml
-        big:
-          title: Big
-          description: Big graphics. Used e.g. in item detail view
-          $ref: types/graphic.schema.yaml
+        usItemClass:
+          title: Item class
+          $ref: types/uint32.schema.yaml
```

Comment:
Doesn't need to happen now, but this could also be changed to a fixed list to be easier to work with.

Reply:
Yes. I just didn't want to break existing mods that modify the `items.json`.

Reply:
It will always be a problem with already released schemas. I guess we could solve that with anyOf and then eventual deprecation.